### ISSUE TITLE
Android support?

### ISSUE THREAD

**Issue by Pablokiryu** [BurningSands70/stremio-downloader#9]
Would it be even possible to implement , do you know?
I would be willing to help in development of both.

**Comment by BurningSands70** [comment]
I don't think it would be possible as the technique used in this project is based on the Web version of Stremio which is not responsive enough for phones.

But the method explained in the comments of this post seems to work well for many users:
https://www.reddit.com/r/StremioAddons/comments/ekwj5x/how_to_download_moviesseries_on_android/

It also seems like a simple enough solution for this requirement on Android.

**Comment by Pablokiryu** [comment]
Thank you for your reply.
